app/views/research/submission_test_run_results/_fail.html.haml

```diff
@@ -1,7 +1,7 @@
-.test-run-result.open.fail
+.test-run-result.fail
```

Comment:
Why has this changed? And the one in `_pass.html.haml`?

Reply:
The JS collapses everything by default. Did this use to work in a way that it was open by default?

Reply:
Yeah. Failed should always be open by default, and passed should always be collapsed by default. So this works to do that by having the different classes in the different files.

Reply:
Okay.